index.html

```diff
@@ -609,8 +664,9 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
             </li>
           </ol>
         </li>
-        <li>If the <var>add to performance entry buffer</var> flag is set, add
-        <var>new entry</var> to the <a>performance entry buffer</a>.
+        <li>If the <a>maxBufferSize</a> for this <var>entryType</var> has not
```

Comment:
This is underdefined. You need to use the entryType to fetch the right tuple from the map, get maxBufferSize from that and then use that.

Reply:
The new version includes these updates to refer to the map.